modules/fbs/fbs.module

```diff
@@ -520,3 +623,22 @@ function _fbs_wayf_login($name, $value = NULL) {
 
   return NULL;
 }
+
+/**
+ * Implements hook_ting_pre_execute().
+ *
+ * Filter out blacklisted branches from search.
+ */
+function fbs_ting_pre_execute($request) {
+  // We only want to modify searches.
+  if (gettype($request) != 'TingClientSearchRequest') {
+    return;
+  }
```

Comment:
You're right: We should target collections too. Don't know what I was thinking about when I wrote that.

I guess the best option is instanceof then: http://stackoverflow.com/questions/3017684/what-is-the-difference-between-is-a-and-instanceof